python/orca/test/bigdl/orca/learn/spark/conftest.py

```diff
@@ -13,13 +13,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
 import pytest
+from pyspark.sql.types import ArrayType, DoubleType
+from pyspark.sql import SparkSession
 
 
 @pytest.fixture(autouse=True, scope='package')
 def orca_context_fixture():
-    from bigdl.orca import init_orca_context
-    sc = init_orca_context(cores=4, spark_log_level="INFO")
-    yield sc
-    sc.stop()
+    from bigdl.orca import init_orca_context, stop_orca_context
+    conf = {"spark.python.worker.reuse": "false"}
```

Comment:
we can remove this and run once to confirm the error?

Reply:
ok, let's try and cry 😢